{{cookiecutter.project_slug}}/home/management/commands/createsuperuserauto.py

```diff
@@ -4,36 +4,41 @@
 
 
 class Command(createsuperuser.Command):
```

Comment:
I think it should be custom command instead of inheriting form this parent command now. There it will just have the email argument, and we get user with that email and make that user superuser. Also, let's add another file  `make_superuser.py` or something like that instead of modifying this one.